docs/01-tooling/03-fast-and-isolated-JS-unit-tests.md

```diff
@@ -2,10 +2,159 @@
 
 ### Intro
 
+For testing frontend code, MediaWiki provides a browser based QUnit setup. For
+running the tests, you have to spin up MediaWiki, and load
```

Comment:
"spin up MediaWiki" may be too light a phrasing for the Vagrant tax we have to pay :]

Reply:
Added a small note about vagrant 😀 

`you have to spin up MediaWiki, usually through vagrant, and`

And added to the section about _a long time to run_:

`There are also big startup costs, for powering up vagrant and the whole system. `